app/api/business/validators/brief_atm_validator.py

```diff
@@ -196,6 +212,19 @@ def validate_required(self):
             )
         if not self.validate_closed_at():
             errors.append('The closing date must be at least 2 days into the future')
+        if not self.validate_closed_at_blackout():
+            blackout_dates = key_values_service.get_by_key('blackout_dates')
```

Comment:
Instead of fetching this data here and in `validate_closed_at_blackout`, do you think we can simplify by getting and parsing the dates at the start of this function?  You could then pass the dictionary or individual dates as parameters to `validate_closed_at_blackout`.

Reply:
I have refactored the code to simplify this process.